spec.bs

```diff
@@ -18,6 +18,13 @@ Default Biblio Status: current
 Markup Shorthands: markdown yes
 </pre>
 
+<pre class="anchors">
+spec: ecma262; urlPrefix: https://tc39.es/ecma262/
+  type: dfn
```

Comment:
You should sort these into their appropriate types. `RegExp` is an `interface`, and `RegExpExec` is an `abstract-op`. That will change how you link to them; you do `{{RegExp}}` and `[$RegExpExec$]`